test/dispatcher.js

```diff
@@ -9,11 +21,10 @@ suite('Event Generation and Dispatching', function() {
     document.body.appendChild(container);
   });
 
-  teardown(function() {
+  after(function() {
     document.body.removeChild(container);
   });
 
-  var pepde = PointerEventsPolyfill.dispatcher.eventSources;
```

Comment:
You can't just reference the module directly here. See above in the capture tests where you have this done correctly.